scripts/update-navigation.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+OUTPUT="/tmp/`uuidgen`"
+RELEASE_BRANCH=${1:-master}
+
+step "Updating mapbox-navigation-ios…"
+git fetch --depth=1 --prune
```

Comment:
I’d just make this a normal `fetch`. Otherwise it could cause the repository to get in a bad state for working with other branches for development purposes later.

Reply:
Whatever works! This is just copied over from the old one (which I don't think ever actually ran).